### ISSUE TITLE
Newer hp laptops having wi-fi issues

### ISSUE THREAD

**Issue by warriormaster12** [digabi/digabi-os#92]
Some hp laptops including mine that use realtek wi-fi component don't have drivers included in the linux kernel. 

The problem can be solved by building this github project [https://h30434.www3.hp.com/t5/Notebook-Wireless-and-Networking/Realtek-8723DE-wifi-module-amp-Bluetooth-Linux-driver/td-p/6477307](url) 
After which just restart the computer and wi-fi wil work without issues.

**Comment by Khadgarcs** [comment]
Supporting newer wifi-cards should also fix it i think. But yea would love this addition, since i've already had to buy 2 wifi adapters for my laptop to attain connection.

**Comment by developerfromjokela** [comment]
It isn't that hard for digabi to include new drivers. All other linux distros are doing it except digabi.
digabi, wake up! We need drivers! We don't want to spend money on wifi adapters because your os is shit!

**Comment by developerfromjokela** [comment]
True, but everyone doesn't have knowledge to do that. That's why digabi has its developers.

**Comment by warriormaster12** [comment]
@developerfromjokela that's true

**Comment by mplattu** [comment]
@warriormaster12 this has been open for a while. As stated in [this blogpost](https://www.ylioppilastutkinto.fi/ajankohtaista/blogitekstit/970-Digitaalinen%20ylioppilastutkinto%20kehittyy%20koronan%20varjossa) we're upgrading the Abitti to Debian Buster. The firmware is [available in the Buster](https://packages.debian.org/buster/firmware-realtek) so there are pretty good chances for getting this to work.

I hope there won't be any COVID-related delays for getting the Buster out. At the moment it looks good as most upper secondaries are operating normally and we will get a lot of feedback before the spring 2021 exams.

**Comment by warriormaster12** [comment]
@warriormaster12 sounds great !! I'll be happy to test the Buster build and provide feedback :D

**Comment by mplattu** [comment]
@warriormaster12 The Buster beta is now out, please give it a try: https://www.abitti.fi/blogi/2020/10/uusi-abitti-versio-beta-testikaytossa/

**Comment by warriormaster12** [comment]
@mplattu alright, I'm going to try and look at the beta release :D

**Comment by warriormaster12** [comment]
@mplattu Sorry for the late response but I was just able to try a couple of versions of new stable abitti release. Network drivers seems to not work on my device at least (my network card is: RTL8723DE)
